modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assembly:
+//
+// * installing/installing-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc_{context}"]
+= Disabling ShipwrightBuilds build controller and webhook pod
+
+You can disable `ShipwrightBuilds` CR without disabling the shared-resources CSI driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
```

Comment:
Can you include what the acronyms CR and CSI stand for? 

From the [Language](https://ccs-internal-documentation.pages.redhat.com/peer-review/#_language) section of the [Red Hat peer review guide](https://ccs-internal-documentation.pages.redhat.com/peer-review/):  "Acronyms are expanded on first use."